loopdb/sqlc/queries/swaps.sql

```diff
@@ -106,11 +114,29 @@ INSERT INTO loopout_swaps (
     max_prepay_routing_fee,
     publication_deadline,
     single_sweep,
-    payment_timeout
+    payment_timeout,
+    is_asset_swap
 ) VALUES (
-    $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12
+    $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13
 );
 
+-- name: InsertLoopOutAsset :exec
+INSERT INTO loopout_swaps_assets (
+    swap_hash,
+    asset_id,
+    swap_rfq_id,
+    prepay_rfq_id
+) VALUES (
+    $1, $2, $3, $4
+);
+
+-- name: UpdateLoopOutAssetOffchainPayments :exec
+UPDATE loopout_swaps_assets
+SET
+    asset_amt_paid_swap = $2,
```

Comment:
Should we add an on-conflict clause?

Reply:
But this will never return an error. I meant that it could return an error if the amounts are already non-zero. It's also not super important but an extra measure that we don't update the row accidentally.